project_templates/fastapi_safir_app/example/src/example/main.py

```diff
@@ -0,0 +1,54 @@
+"""The main application factory for the example service.
+
+Notes
+-----
+Be aware that, following the normal pattern for FastAPI services, the app is
+constructed when this module is loaded and is not deferred until a function is
+called.
+"""
+
+from importlib.metadata import metadata
+
+from fastapi import FastAPI
+from safir.dependencies.http_client import http_client_dependency
+from safir.logging import configure_logging
+from safir.middleware.x_forwarded import XForwardedMiddleware
+
+from .config import config
+from .handlers.external import external_router
+from .handlers.internal import internal_router
+
+__all__ = ["app", "config"]
+
+
+configure_logging(
+    profile=config.profile,
+    log_level=config.log_level,
+    name=config.logger_name,
+)
+
+app = FastAPI()
+"""The main FastAPI application for example."""
+
+# Define the external routes in a subapp so that it will serve its own OpenAPI
+# interface definition and documentation URLs under the external URL.
+_subapp = FastAPI(
+    title="example",
+    description="Short one-sentence summary of the app",
```

Comment:
This is totally fine, though I guess you could also use the `metadata` API to grab the summary from `setup.cfg`?

Reply:
I went back and forth on this because I wasn't sure which place people were more likely to update. I'm going to leave this for now but take a look at this when investigating creating the app in a function instead.